reference/resources/bgpconfig.md

```diff
@@ -46,7 +46,7 @@ spec:
 | name     | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, or `-`. | string |
 
 - The resource with the name `default` has a specific meaning - this contains the BGP global default configuration.
-- The resources with the name `node.<nodename>` contain the node-specific overrides, and will be applied to the node `<nodename>`. When deleting a node the BGPConfiguration resource associated with the node will also be deleted.
+- The resources with the name `node.<nodename>` contain the node-specific overrides, and will be applied to the node `<nodename>`. When deleting a node the BGPConfiguration resource associated with the node will also be deleted. Only prefixAdvertisements, listenPort, logSeverityScreen and Extensions can be overridden this way.
```

Comment:
What Extensions?  That doesn't seem to be in the table?

Reply:
Ah right, I guess I was comparing against the private doc page.  Extensions are private-only.